### PR TITLE
Revise README

### PR DESCRIPTION
**Pull request by joelnitta**
I split out most of the content of the README into separate, smaller MDs. Most of these are explanations of tasks that only maintainers need to carry out, like importing a lesson for the first time, or generating the translated lesson webpage. I think this will make it easier for volunteers to contribute, since they won't feel like they need to read the whole thing.

It we want to move the various smaller MDs to the wiki, I think that would fine as well. I just wanted to do it this way first so we all have a chance to see the proposed changes.

I also added the `local/` folder to `.gitignore`, since this gets automatically generated by `po4gitbook` and we don't want translators accidentally committing it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,192 +2,43 @@
...
-Download Git for Windows (Git-Bash) by following the instructions here:
-
-https://gitforwindows.org/
-
-or
-
-https://git-scm.com/
-
-- Mac OS
+We are translating (and keeping up-to-date) the Software Carpentry lessons, not revising original lesson material.
```
Comment (TomKellyGenetics) [inline README.md:17]: This is an important point, this should be emphasised. Perhaps we need to put a similar notice in the README for translated lesson repos?

File: README.md [hunk 2]
```diff
@@ -201,241 +52,58 @@ git clone git@github.com:GitHubUser/i18n.git
...
-If you already have a fork of translation repository, please pull changes for the current
-   version from the organisation repository:
+3. Populate submodules.
 
 ```
-git checkout ja
-git remote add swc-ja git@github.com:swcarpentry-ja/i18n.git
-git pull swc-ja ja
+git submodule init
+git submodule update
```
Comment (TomKellyGenetics) [inline README.md:61]:
```suggestion
git submodule update --remote
```

File: README.md [hunk 3]
```diff
@@ -201,241 +52,58 @@ git clone git@github.com:GitHubUser/i18n.git
...
-Alternatively, you can update the submodules for every lesson:
-
-```
-git submodule foreach git pull origin master
-```
-
-4. Create translation files for the updated version of the lessons
-
-Run `po4gitbook/update.sh` - That creates/updates the `po` directory with the `.pot` files to use in translations.
+Repeat steps 4 and 5 until the PR passes review.
```
Comment (TomKellyGenetics) [inline README.md:81]:
```suggestion
Repeat steps 4 and 5 until the PR passes review. Don't hesitate to push contributions to your
user directory. These can be added to an open pull request and we can make revisions during review.
```

```

File: README.md [hunk 4]
```diff
@@ -201,241 +52,58 @@ git clone git@github.com:GitHubUser/i18n.git
 cd i18n
 ```
```
Comment (TomKellyGenetics) [inline README.md:56]: Are we missing step 2? I think we need to clone before loading submodules.

```suggestion
2. Clone this repository from your personal account (e.g., GitHubUser). This is your local copy to manage your version of
 the translation files.

cd directory
git clone git@github.com:GitHubUser/i18n.git
cd i18n
```

**Review by TomKellyGenetics — APPROVED** [review]
Great idea. It is getting very long which could be daunting for newcomers.

File: README.md (hunk 4 above)
Comment (TomKellyGenetics) [reply README.md:56]: Sorry I can't get MD backticks into suggestion mode

File: README.md (hunk 1 above)
Comment (joelnitta) [reply README.md:17]: OK let's keep that in mind. It doesn't require further edits to this README though.

File: README.md (hunk 4 above)
Comment (joelnitta) [reply README.md:56]: Actually, it's not missing in the README in my PR. You can see it if you look at the actual file instead of the diff.

Comment (TomKellyGenetics) [reply README.md:56]: Ah yes, it was the diff that confused me.

File: README.md (hunk 1 above)
Comment (TomKellyGenetics) [reply README.md:17]: Yeah it should be much clearer in a shorter README.

**Comment by TomKellyGenetics** [comment]
I vote to merge this soon as we have new contributors interested to get involved:
https://github.com/swcarpentry-i18n/i18n/pull/13

**Review by TomKellyGenetics — APPROVED** [review]
Thanks for addressing this so quickly (see #44). @megane0-0 Thanks for helping out, we need feedback from new contributors to make it easier to get involved. :)

**Comment by TomKellyGenetics** [comment]
I think we should still agree as a group to merge changes to the lessons but I'm going to go ahead and merge these kind of meta-data things that are useful for new translators.

**Pull request merged**